Widgets/Widgets/VarmintWidgetButton.cs

```diff
@@ -47,25 +45,27 @@ private EventHandledState Ignore_OnFlick(VarmintWidget arg1, Vector2 arg2, Vecto
         //--------------------------------------------------------------------------------------
         private void Render(GameTime gameTime, VarmintWidget widget)
         {
-            var textToDisplay = (Content == null) ? "" : Content.ToString();
+            if (InhibitRendering) { return; }
```

Comment:
If isVisible is false, then this method will not get invoked, so you can remove this